adcc/workflow.py

```diff
@@ -191,13 +192,22 @@ def run_adc(data_or_matrix, n_states=None, kind="any", conv_tol=None,
     if eigensolver is None:
         eigensolver = "davidson"
 
+    # Setup solvent coupling terms and energy corrections
+    solvent_matrix_terms, solvent_energy_corrections = \
+        setup_solvent(matrix, solvent_scheme)
```

Comment:
in these cases I prefer
```suggestion
    ret = setup_solvent(matrix, solvent_scheme)
    solvent_matrix_terms, solvent_energy_corrections = ret
```

Reply:
Done.